joblib/test/test_parallel.py

```diff
@@ -278,6 +297,31 @@ def test_dispatch_multiprocessing():
         yield check_dispatch_multiprocessing, backend
 
 
+def test_batching_auto_threading():
+    # batching='auto' with the threading backend leaves the effective batch
+    # size to 1 (no batching) as it has been found to never be beneficial with
+    # this low-overhead backend.
+    p = Parallel(n_jobs=2, batch_size='auto', backend='threading')
+    p(delayed(id)(i) for i in range(5000))  # many very fast tasks
+    assert_equal(p._effective_batch_size, 1)
+
+
+def test_batching_auto_multiprocessing():
+    # Batching is not enabled whith the threading backend as it has found
```

Comment:
I guess this comment is a copy and paste of the one in test_batching_auto_threading and should be removed.